src/status_im/ui/screens/home/sheet/views.cljs

```diff
@@ -57,5 +59,36 @@
    [invite/list-item
     {:accessibility-label :chats-menu-invite-friends-button}]])
 
+(defn add-new-view2 []
+  [react/view
+   [quo/list-item
+    {:theme                        :main
+     :title                        (i18n/label :t/new-chat)
+     :icon-bg-color                :transparent
+     :icon-container-style         {:padding-horizontal 0}
+     :container-padding-horizontal {:padding-horizontal 4}
+     :container-padding-vertical   12
+     :title-column-style           {:margin-left 2}
+     :icon-color                   (if (theme/dark?) colors/neutral-40 colors/neutral-50)
+     :accessibility-label          :start-a-new-chat
+     :icon                         :main-icons2/new-message
+     :on-press                     #(hide-sheet-and-dispatch [:open-modal :new-chat])}]
+   [quo/list-item
+    {:theme                        :main
+     :title                        (i18n/label :t/add-a-contact)
+     :icon-bg-color                :transparent
+     :icon-container-style         {:padding-horizontal 0}
+     :container-padding-horizontal {:padding-horizontal 4}
+     :container-padding-vertical   12
+     :title-column-style           {:margin-left 2}
+     :icon-color                   (if (theme/dark?) colors/neutral-40 colors/neutral-50)
+     :accessibility-label          :add-a-contact
+     :subtitle                     (i18n/label :t/enter-a-chat-key)
+     :icon                         :main-icons2/add-user
+     :on-press                     #(hide-sheet-and-dispatch [:open-modal :new-contact])}]])
+
+(def add-new2
+  {:content add-new-view2})
+
 (def add-new
   {:content add-new-view})
```

Comment:
We also shouldn't be editing `ui`

Reply:
I had to do this to integrate this view on the old ui .. Will migrate all of it to ui2 after the series of the issues are done.
Thanks Omar